docs/self-service/README.md

```diff
@@ -223,3 +223,58 @@ DEV_IMG ?= # <== NOTE:  outside dev, change this!!!
+    kube-system               coredns-fb8b8dccf-hhc4b                      1/1     Running   0          5d1h
+    kube-system               etcd-kind-control-plane                      1/1     Running   0          5d1h
+    kube-system               ip-masq-agent-m5jkt                          1/1     Running   0          5d1h
+    kube-system               kindnet-5s6tz                                1/1     Running   1          5d1h
+    kube-system               kube-apiserver-kind-control-plane            1/1     Running   0          5d1h
+    kube-system               kube-controller-manager-kind-control-plane   1/1     Running   0          5d1h
+    kube-system               kube-proxy-x57n5                             1/1     Running   0          5d1h
+    kube-system               kube-scheduler-kind-control-plane            1/1     Running   0          5d1h
+    vsphere-provider-system   vsphere-provider-controller-manager-0        1/1     Running   0          5d1h
+    ```
+2. If any are failing then view the pod logs to review errors: 
+    ```shell
+    $> kubectl logs <pod name> --namespace <pod namespace>
+    ```
+3. After the bootstrap pods have been created, vSphere will create one or more VMs in accordance to the machine yaml files specified with the clusterctl create cluster command.
+4. Should the clusterctl create cluster command fail to retrieve the admin.conf file the following steps can be used:
+	1. Connect to the manager pod in the bootstrap cluster: 
+    ```shell
+    $> kubectl exec vsphere-provider-controller-manager-0 -it /bin/bash --namespace vsphere-provider-system
+    ```
+	2. SSH on the provisioned master VM within vSphere from the manager pod: 
+    ```shell
+    $> ssh -i ~/.ssh/vsphere_tmp ubuntu@<vm ip address>
+    ```
+	3. Verify if the following file exists: `/etc/kuberenetes/admin.conf`.  Please note it may take a couple minutes for cloud-init to process and create these files.
+        ```shell
+        $> ls /etc/kubernetes/
+        admin.conf  cloud-config  controller-manager.conf  kubeadm_config.yaml  kubelet.conf  manifests  pki  scheduler.conf
+        ```
+	4. If either the file or folder do not exist then check the following log files for failed commands: `/var/log/cloud-init.log` and `/var/log/cloud-init-output.log`.
+	5. If the log files are still being appended to then cloud-init has not finished processing and may need more time to run.
+	6. An example failure which may be listed in `/var/log/cloud-init.log` is `2019-05-06 18:22:41,691 - util.py[WARNING]: Failed loading yaml blob. unacceptable character #xdccf: special characters are not allowed`.  This error indicates an incorrect entry in machines.yaml or machineset.yaml which was specified in the `clusterctl create cluster` command.  Commonly this could be leaving in the `- xxxx` values in the machines.yaml for sections such as DNS and trustedCerts.
```

Comment:
```suggestion
	6. An example failure which may be listed in `/var/log/cloud-init.log` is `2019-05-06 18:22:41,691 - util.py[WARNING]: Failed loading yaml blob. unacceptable character #xdccf: special characters are not allowed`.  This error indicates an incorrect entry in `machines.yaml` or `machineset.yaml` which was specified in the `clusterctl create cluster` command.  Commonly this could be leaving in the `- xxxx` values in the `machines.yaml` for sections such as `DNS` and `trustedCerts`.
```